src/SeleniumLibrary/keywords/element.py

```diff
@@ -56,35 +56,54 @@ def element_should_contain(self, locator, expected, message=None):
         The ``message`` argument can be used to override the default error
         message.
 
+        The ``ignore_case`` argument can be set to True to compare case
+        insensitive, default is False.
+
         Use `Element Text Should Be` if you want to match the exact text,
         not a substring.
```

Comment:
In new line, add text: ``ignore_case`` argument is new in SeleniumLibrary 3.1.

Please fix in keywords where the new argument is added.